frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -229,7 +242,11 @@ export const Onboarding = () => {
                 order={2}
                 mb="lg"
               >{t`Set up your ${applicationName}`}</Title>
-              <Accordion.Item value="database" data-testid="database-item">
+              <Accordion.Item
+                value="database"
+                data-testid="database-item"
+                ref={itemRefs["database"]}
```

Comment:
No need to use object map notation here (this applies to other refs in this component too):

```suggestion
                ref={itemRefs.database}
```

Reply:
I know there's no _need_ for it, but isn't it more readable to have it uniform since we already have to use this notation for `["x-rays"]`?